src/vue/pages/VerificationCorporate.vue

```diff
@@ -242,17 +241,14 @@ export default {
     }
     if (this.isFormPopulatable) {
       this.form = this.parseKycData(this.kycLatestData)
-
-      if (!this.isFormEditable) {
-        this.disableForm()
-      }
```

Comment:
If I was not mistaken in previous comments, form still not editable if the request has "pending" status. we should modify `isFormEditable` but not delete it

Reply:
The form is editable. I checked it for all states (pending/approved/rejected).